source/command-monitoring/command-monitoring.rst

```diff
@@ -250,7 +250,10 @@ value MUST be replaced with an empty BSON document. The list is as follows:
    * - ``copydbgetnonce``
    * - ``copydbsaslstart``
    * - ``copydb``
-   * - ``isMaster`` or ``ismaster`` when ``speculativeAuthenticate`` is present
+   * - ``hello`` or legacy hello, regardless of whether ``speculativeAuthenticate`` is present
```

Comment:
the previous wording was a little ambiguous but my reading was that we only redact the body of the command (with either capitalization) when speculativeAuthenticate is present. 

this seems to change that, so that we redact the body of this command regardless of if speculativeAuthenticate is present.

is that intentional?

Reply:
That is ambiguous. I parsed it the other way as I glossed over the capitalization difference in `isMaster`. I think you're right that the intention is to redact only when `spectulativeAuthenticate` is present. I'll change that.